[2023-12-18 19:00:02,225] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-18T18:00:00+00:00 [queued]>
[2023-12-18 19:00:02,297] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: dag_with_python_weather_v01.fill 2023-12-18T18:00:00+00:00 [queued]>
[2023-12-18 19:00:02,298] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-12-18 19:00:02,299] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2023-12-18 19:00:02,300] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-12-18 19:00:02,326] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): fill> on 2023-12-18T18:00:00+00:00
[2023-12-18 19:00:02,347] {standard_task_runner.py:52} INFO - Started process 17871 to run task
[2023-12-18 19:00:02,357] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_with_python_weather_v01', 'fill', '2023-12-18T18:00:00+00:00', '--job-id', '157', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_weather.py', '--cfg-path', '/tmp/tmpbn7n0vmm', '--error-file', '/tmp/tmpnzmunwiq']
[2023-12-18 19:00:02,366] {standard_task_runner.py:77} INFO - Job 157: Subtask fill
[2023-12-18 19:00:02,526] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_with_python_weather_v01.fill 2023-12-18T18:00:00+00:00 [running]> on host 5f3722f1fc3b
[2023-12-18 19:00:02,585] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ggoni
AIRFLOW_CTX_DAG_ID=dag_with_python_weather_v01
AIRFLOW_CTX_TASK_ID=fill
AIRFLOW_CTX_EXECUTION_DATE=2023-12-18T18:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-12-18T18:00:00+00:00
[2023-12-18 19:00:05,263] {python.py:118} INFO - Done. Returned value was: None
[2023-12-18 19:00:05,279] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=dag_with_python_weather_v01, task_id=fill, execution_date=20231218T180000, start_date=20231218T190002, end_date=20231218T190005
[2023-12-18 19:00:05,318] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-12-18 19:00:05,348] {local_task_job.py:146} INFO - Task exited with return code 0
